packages/e2e-tests/helpers.ts

```diff
@@ -149,12 +149,16 @@ export async function warpToMessage(screen: Screen, text: string, line?: number)
 }
 
 export async function executeInConsole(screen: Screen, text: string) {
-  // TODO [PR 7550]
+  const consoleTextArea = screen.locator('[data-test-id="ConsoleRoot"] textarea');
+
+  await consoleTextArea.focus();
+  await consoleTextArea.fill(text);
+  await consoleTextArea.press("Enter");
 }
 
 export async function checkMessageObjectContents(
   screen: Screen,
-  message: HTMLElement,
+  message: ElementHandle<HTMLElement | SVGElement>,
```

Comment:
Interesting! 😮

Reply:
Ah, this was from me trying to fill out this helper it's what playwright returns from `waitForSelector`.